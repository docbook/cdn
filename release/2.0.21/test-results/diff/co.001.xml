<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: co.001</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: co.001</h2>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">John</span>
                     <span class="surname">Himpel</span>
                  </span>
               </h3>
            </div>
         </header>
         <p deltaxml:deltaV2="A">Callouts inside <code class="tag-element">userinput</code> elements.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><pre deltaxml:deltaV2="A" class="screen language-none">
            <code>
               <code class="prompt">$</code>
               <strong class="userinput">
                  <code>cp a b<a href="#cp" class="co" id="cp-co">
                        <span data-number="1" class="callout-bug callout-bug-unicode">❶</span>
                     </a>
                  </code>
               </strong>
               <code class="prompt">$</code>
               <strong class="userinput">
                  <code>mv a b<a href="#mv" class="co" id="mv-co">
                        <span data-number="2" class="callout-bug callout-bug-unicode">❷</span>
                     </a>
                  </code>
               </strong>
            </code>
         </pre><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">calloutlist</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
            <dl deltaxml:deltaV2="A">
               <dt id="cp" class="callout">
                  <a href="#cp-co" class="callout-link">
                     <span data-number="1" class="callout-bug callout-bug-unicode">❶</span>
                  </a> </dt>
               <dd>
                  <p>This is a copy.</p>
               </dd>
               <dt id="mv" class="callout">
                  <a href="#mv-co" class="callout-link">
                     <span data-number="2" class="callout-bug callout-bug-unicode">❷</span>
                  </a> </dt>
               <dd>
                  <p>This is a move.</p>
               </dd>
            </dl><p deltaxml:deltaV2="B">Callouts inside <code class="tag-element">userinput</code> elements.</p><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><pre deltaxml:deltaV2="B" class="screen language-none">
               <code>
                  <code class="prompt">$</code>
                  <strong class="userinput">
                     <code>cp a b<a href="#cp" class="co" id="cp-co">
                           <span data-number="1" class="callout-bug callout-bug-unicode">❶</span>
                        </a>
                     </code>
                  </strong>
                  <code class="prompt">$</code>
                  <strong class="userinput">
                     <code>mv a b<a href="#mv" class="co" id="mv-co">
                           <span data-number="2" class="callout-bug callout-bug-unicode">❷</span>
                        </a>
                     </code>
                  </strong>
               </code>
            </pre><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="calloutlist">
               <dl>
                  <dt id="cp" class="callout">
                     <a href="#cp-co" class="callout-link">
                        <span data-number="1" class="callout-bug callout-bug-unicode">❶</span>
                     </a> </dt>
                  <dd>
                     <p>This is a copy.</p>
                  </dd>
                  <dt id="mv" class="callout">
                     <a href="#mv-co" class="callout-link">
                        <span data-number="2" class="callout-bug callout-bug-unicode">❷</span>
                     </a> </dt>
                  <dd>
                     <p>This is a move.</p>
                  </dd>
               </dl>
            </div>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>