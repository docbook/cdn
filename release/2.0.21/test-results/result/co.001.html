<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for co.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: co.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">6</span> differences)</th><th>Actual (<span class="adiff">6</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: co.001&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: co.001&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;John&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Himpel&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;Callouts inside <span class="tag">&lt;code <span class="att">class="tag-element"</span>&gt;userinput&lt;/code&gt;</span> elements.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;pre <span class="att">class="screen language-none"</span>&gt;
            <span class="tag">&lt;code&gt;
               <span class="tag">&lt;code <span class="att">class="prompt"</span>&gt;$&lt;/code&gt;</span>
               <span class="tag">&lt;strong <span class="att">class="userinput"</span>&gt;
                  <span class="tag">&lt;code&gt;cp a b<span class="tag">&lt;a <span class="att">class="co"</span> <span class="att">href="#cp"</span> <span class="att">id="cp-co"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="1"</span>&gt;❶&lt;/span&gt;</span>
                     &lt;/a&gt;</span>
                  &lt;/code&gt;</span>
               &lt;/strong&gt;</span>
               <span class="tag">&lt;code <span class="att">class="prompt"</span>&gt;$&lt;/code&gt;</span>
               <span class="tag">&lt;strong <span class="att">class="userinput"</span>&gt;
                  <span class="tag">&lt;code&gt;mv a b<span class="tag">&lt;a <span class="att">class="co"</span> <span class="att">href="#mv"</span> <span class="att">id="mv-co"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="2"</span>&gt;❷&lt;/span&gt;</span>
                     &lt;/a&gt;</span>
                  &lt;/code&gt;</span>
               &lt;/strong&gt;</span>
            &lt;/code&gt;</span>
         &lt;/pre&gt;</span><span class="diff txtchg">
         </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">calloutlist</span>"</span>&gt;
            <span class="tag diff del">&lt;dl&gt;
               <span class="tag">&lt;dt <span class="att">class="callout"</span> <span class="att">id="cp"</span>&gt;
                  <span class="tag">&lt;a <span class="att">class="callout-link"</span> <span class="att">href="#cp-co"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="1"</span>&gt;❶&lt;/span&gt;</span>
                  &lt;/a&gt;</span> &lt;/dt&gt;</span>
               <span class="tag">&lt;dd&gt;
                  <span class="tag">&lt;p&gt;This is a copy.&lt;/p&gt;</span>
               &lt;/dd&gt;</span>
               <span class="tag">&lt;dt <span class="att">class="callout"</span> <span class="att">id="mv"</span>&gt;
                  <span class="tag">&lt;a <span class="att">class="callout-link"</span> <span class="att">href="#mv-co"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="2"</span>&gt;❷&lt;/span&gt;</span>
                  &lt;/a&gt;</span> &lt;/dt&gt;</span>
               <span class="tag">&lt;dd&gt;
                  <span class="tag">&lt;p&gt;This is a move.&lt;/p&gt;</span>
               &lt;/dd&gt;</span>
            &lt;/dl&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: co.001&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: co.001&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;John&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Himpel&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span>&gt;
            <span class="tag diff add">&lt;p&gt;Callouts inside <span class="tag">&lt;code <span class="att">class="tag-element"</span>&gt;userinput&lt;/code&gt;</span> elements.&lt;/p&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;pre <span class="att">class="screen language-none"</span>&gt;
               <span class="tag">&lt;code&gt;
                  <span class="tag">&lt;code <span class="att">class="prompt"</span>&gt;$&lt;/code&gt;</span>
                  <span class="tag">&lt;strong <span class="att">class="userinput"</span>&gt;
                     <span class="tag">&lt;code&gt;cp a b<span class="tag">&lt;a <span class="att">class="co"</span> <span class="att">href="#cp"</span> <span class="att">id="cp-co"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="1"</span>&gt;❶&lt;/span&gt;</span>
                        &lt;/a&gt;</span>
                     &lt;/code&gt;</span>
                  &lt;/strong&gt;</span>
                  <span class="tag">&lt;code <span class="att">class="prompt"</span>&gt;$&lt;/code&gt;</span>
                  <span class="tag">&lt;strong <span class="att">class="userinput"</span>&gt;
                     <span class="tag">&lt;code&gt;mv a b<span class="tag">&lt;a <span class="att">class="co"</span> <span class="att">href="#mv"</span> <span class="att">id="mv-co"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="2"</span>&gt;❷&lt;/span&gt;</span>
                        &lt;/a&gt;</span>
                     &lt;/code&gt;</span>
                  &lt;/strong&gt;</span>
               &lt;/code&gt;</span>
            &lt;/pre&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;div <span class="att">class="calloutlist"</span>&gt;
               <span class="tag">&lt;dl&gt;
                  <span class="tag">&lt;dt <span class="att">class="callout"</span> <span class="att">id="cp"</span>&gt;
                     <span class="tag">&lt;a <span class="att">class="callout-link"</span> <span class="att">href="#cp-co"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="1"</span>&gt;❶&lt;/span&gt;</span>
                     &lt;/a&gt;</span> &lt;/dt&gt;</span>
                  <span class="tag">&lt;dd&gt;
                     <span class="tag">&lt;p&gt;This is a copy.&lt;/p&gt;</span>
                  &lt;/dd&gt;</span>
                  <span class="tag">&lt;dt <span class="att">class="callout"</span> <span class="att">id="mv"</span>&gt;
                     <span class="tag">&lt;a <span class="att">class="callout-link"</span> <span class="att">href="#mv-co"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="2"</span>&gt;❷&lt;/span&gt;</span>
                     &lt;/a&gt;</span> &lt;/dt&gt;</span>
                  <span class="tag">&lt;dd&gt;
                     <span class="tag">&lt;p&gt;This is a move.&lt;/p&gt;</span>
                  &lt;/dd&gt;</span>
               &lt;/dl&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: co.001</h2><div class="author"><h3><span class="personname"><span class="firstname">John</span> <span class="surname">Himpel</span></span></h3></div></header><p>Callouts inside <code class="tag-element">userinput</code> elements.</p><pre class="screen language-none"><code><code class="prompt">$</code><strong class="userinput"><code>cp a b<a href="#cp" class="co" id="cp-co"><span data-number="1" class="callout-bug callout-bug-unicode">❶</span></a></code></strong>
<code class="prompt">$</code><strong class="userinput"><code>mv a b<a href="#mv" class="co" id="mv-co"><span data-number="2" class="callout-bug callout-bug-unicode">❷</span></a></code></strong></code></pre><div class="calloutlist"><dl><dt id="cp" class="callout"><a href="#cp-co" class="callout-link"><span data-number="1" class="callout-bug callout-bug-unicode">❶</span></a> </dt><dd><p>This is a copy.</p></dd><dt id="mv" class="callout"><a href="#mv-co" class="callout-link"><span data-number="2" class="callout-bug callout-bug-unicode">❷</span></a> </dt><dd><p>This is a move.</p></dd></dl></div></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: co.001</h2><div class="author"><h3><span class="personname"><span class="firstname">John</span> <span class="surname">Himpel</span></span></h3></div></header><div class="content"><p>Callouts inside <code class="tag-element">userinput</code> elements.</p><pre class="screen language-none"><code><code class="prompt">$</code><strong class="userinput"><code>cp a b<a href="#cp" class="co" id="cp-co"><span data-number="1" class="callout-bug callout-bug-unicode">❶</span></a></code></strong>
<code class="prompt">$</code><strong class="userinput"><code>mv a b<a href="#mv" class="co" id="mv-co"><span data-number="2" class="callout-bug callout-bug-unicode">❷</span></a></code></strong></code></pre><div class="calloutlist"><dl><dt id="cp" class="callout"><a href="#cp-co" class="callout-link"><span data-number="1" class="callout-bug callout-bug-unicode">❶</span></a> </dt><dd><p>This is a copy.</p></dd><dt id="mv" class="callout"><a href="#mv-co" class="callout-link"><span data-number="2" class="callout-bug callout-bug-unicode">❷</span></a> </dt><dd><p>This is a move.</p></dd></dl></div></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/co.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>