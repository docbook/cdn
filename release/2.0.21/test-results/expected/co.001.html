<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: co.001</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: co.001</h2><div class="author"><h3><span class="personname"><span class="firstname">John</span> <span class="surname">Himpel</span></span></h3></div></header><p>Callouts inside <code class="tag-element">userinput</code> elements.</p><pre class="screen language-none"><code><code class="prompt">$</code><strong class="userinput"><code>cp a b<a href="#cp" class="co" id="cp-co"><span data-number="1" class="callout-bug callout-bug-unicode">❶</span></a></code></strong>
<code class="prompt">$</code><strong class="userinput"><code>mv a b<a href="#mv" class="co" id="mv-co"><span data-number="2" class="callout-bug callout-bug-unicode">❷</span></a></code></strong></code></pre><div class="calloutlist"><dl><dt id="cp" class="callout"><a href="#cp-co" class="callout-link"><span data-number="1" class="callout-bug callout-bug-unicode">❶</span></a> </dt><dd><p>This is a copy.</p></dd><dt id="mv" class="callout"><a href="#mv-co" class="callout-link"><span data-number="2" class="callout-bug callout-bug-unicode">❷</span></a> </dt><dd><p>This is a move.</p></dd></dl></div></article><script src="../../resources/js/prism.js"></script></body></html>